device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/CalendarService.kt

```diff
@@ -30,14 +31,18 @@ import com.builttoroam.devicecalendar.models.Calendar
 import com.builttoroam.devicecalendar.models.Event
 import io.flutter.plugin.common.MethodChannel
 import com.google.gson.Gson
+import android.provider.CalendarContract.Events
+import android.content.ContentValues
+import java.util.*
 
 
 public class CalendarService : PluginRegistry.RequestPermissionsResultListener {
 
     private val REQUEST_CODE_RETRIEVE_CALENDARS = 0;
     private val REQUEST_CODE_RETRIEVE_EVENTS = 1;
```

Comment:
Can we make these dependent on each other eg
private val REQUEST_CODE_RETRIEVE_EVENTS = REQUEST_CODE_RETRIEVE_CALENDARS + 1;

Reply:
I mean, it doesn't matter what value it will have but sure

Reply:
@nickrandolph shouldn't we prefer to make these a const or enum though? not sure what kind of behaviour your suggestion will have as well when it comes to Java code that gets generated. saw an article on how it translates the definition of constants https://blog.egorand.me/where-do-i-put-my-constants-in-kotlin/

Reply:
There's probably arguments for and against different structures and their relative perf (your link is very focused on how it translates to java rather than the intent of the language). See https://discuss.kotlinlang.org/t/constants-class/2589/2 and https://kotlinlang.org/docs/reference/object-declarations.html for more information on structuring const values
Also, I think making them derived values shouldn't preclude them from being constants. Definitely in C# you can do this and the compiler works out that they're still constants after compilation.